docs/conf.py.in

```diff
@@ -81,8 +81,11 @@ project = u'@PROJECT_NAME@'
 copyright = u'Pelagicore AB, licensed under CC-BY-SA 4.0'
 author = u'Pelagicore AB'
 
-version = u'@PROJECT_VERSION@-alpha'
-release = u"@PROJECT_VERSION@-alpha-@REVISION@ @DATE@"
+# version should be set to @PROJECT_VERSION@ once stable PELUX version is released
+# PROJECT_VERSION is defined in CMakeLists.txt as VERSION
```

Comment:
There is no such thing, as far as I know. The is a 'cmake_minimum_version' which specifies the cmake version, but the doc version is only here.

Reply:
Here: https://github.com/Pelagicore/software-factory/blob/master/CMakeLists.txt

Reply:
Sorry about the mistake I must have checked a CMakeLists which wasn't in the base dir.